openapiart/openapiartgo.py

```diff
@@ -642,12 +642,32 @@ def _build_api_interface(self):
             // httpConnect builds up a http connection
             func (api *{internal_struct_name}) httpConnect() error {{
                 if api.httpClient.client == nil {{
-                    var verify = !api.http.verify
+                    tr := http.Transport{{
+                        DialTLSContext: func(ctx context.Context, network, addr string) (net.Conn, error) {{
+                            tcpConn, err := (&net.Dialer{{}}).DialContext(ctx, network, addr)
+                            if err != nil {{
+                                return nil, err
+                            }}
+                            tlsConn := tls.Client(tcpConn, &tls.Config{{InsecureSkipVerify: !api.http.verify}})
+                            err = tlsConn.Handshake()
+                            if err != nil {{
+                                return nil, err
+                            }}
+                            api.http.conn = tcpConn
+                            return tlsConn, nil
+                        }},
+                        DialContext: func(ctx context.Context, network, addr string) (net.Conn, error) {{
```

Comment:
we shouldn't need this right ? let's remove it.

Reply:
let me check if we remove this will our tests work properly, as the test servers runs on http

Reply:
If removed the UTs are failing. as test server is serving http responses.